frontend/src/libs/components/toggle/toggle.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "src/assets/css/styles.scss";
+
+.toggle {
+  --size: 20px;
+
+  width: calc(var(--size) * 2);
```

Comment:
Why do we need that ?

Reply:
again, to understand the idea of how the shape sizes and dimensions are related to each other, We can adjust --size and it will automatically recalculate the shape of the toggle.
For example by looking the formula of width and height you could easily say that width is twice as large as height, and looking down the code you would see how the toggle circle is calculated based on the size.

But maybe you would agree me to use sass variables instead?